src/widgets/calendar/CalendarDay.tsx

```diff
@@ -22,12 +24,24 @@ export const CalendarDay = ({ date, medias }: CalendarDayProps) => {
       withinPortal
       radius="lg"
       shadow="sm"
-      transition="pop"
+      transitionProps={{
+        transition: 'pop',
+      }}
       onClose={close}
       opened={opened}
     >
       <Popover.Target>
-        <Box onClick={open}>
+        <Box
+          onClick={open}
+          style={{
```

Comment:
Use ``sx`` instead of ``style`` and remove the above hook

Reply:
Not sure if this has any real impact / reason to use it but I'll make the change

Reply:
No impact, but you can simply retrieve the theme without having ``useMantineTheme``